### PR TITLE
1.12.1

### PR DESCRIPTION
**Pull request by zjma**

### CODE REVIEW

File: metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.12",
```
Comment (sherry-x) [inline metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json:2]: update the title to 1.12.1

File: metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.12",
```
Comment (sherry-x) [inline metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json:3]: update description to match the changes

File: metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json (hunk 1 above)
Comment (zjma) [reply metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json]: fixed

File: metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json (hunk 2 above)
Comment (zjma) [reply metadata/2024-05-31-v1.12.1/step_2_upgrade_framework.json]: fixed

**Review by sherry-x — APPROVED** [review]

**Pull request merged**